recipe/bld.bat

```diff
@@ -1,7 +1,8 @@
 :: Build step
 mkdir build_libjpeg
 cd  build_libjpeg
-cmake -G "NMake Makefiles" ^
+
+cmake -G"Ninja" ^
 	-D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
 	-D CMAKE_BUILD_TYPE=Release ^
 	-D ENABLE_STATIC=1 ^
```

Comment:
Is it fine to have static and shared at the same time?

Reply:
yep, and it's what we've got already https://conda-metadata-app.streamlit.app/?q=pkgs%2Fmain%2Fosx-arm64%2Flibjpeg-turbo-2.0.0-h1a28f6b_0.conda